src/headless/converse-chatboxes.js

```diff
@@ -31,10 +32,11 @@ converse.plugins.add('converse-chatboxes', {
             'privateChatsAutoJoined'
         ]);
 
-        let msg_counter = 0;
+        let msg_counter = 0, favicon = new Favico({type : 'circle', position: 'up', animation: 'none'});
```

Comment:
The tests are failing because you're using `let` to instantiate an unchanging variable `favicon`.

Please use `const` instead.